ui/oopsyraidsy/data/06-ew/trial/asura.ts

```diff
@@ -6,7 +6,35 @@ export type Data = OopsyData;
 
 const triggerSet: OopsyTriggerSet<Data> = {
   zoneId: ZoneId.TheGildedAraya,
-  damageWarn: {},
+  damageWarn: {
+    'Asura Chakra Ring 1': '8C9B', // ring AOE
+    'Asura Chakra Ring 2': '8C9C', // ring AOE
+    'Asura Chakra Ring 3': '8C9D', // ring AOE
+    'Asura Chakra Ring 4': '8C9E', // ring AOE
+    'Asura Chakra Ring 5': '8C9F', // ring AOE
+    'Asura Laceration': '8C97', // clone puddle AOEs
+    'Asura Pedestal Purge': '8C81', // circle AOE under boss
+    'Asura Wheel of Deincarnation': '8C83', // donut AOE under boss
+    'Asura Bladewise': '8C85', // line cleave from boss
+    'Asura Image - Pedestal Purge': '8C82', // big circle AOE under image
+    'Asura Image - Wheel of Deincarnation': '8C84', // big donut AOE under image
+    'Asura Image - Bladewise': '8C86', // big line cleave from image
+    'Asura Khadga 1': '8C89', // half-room cleave
+    'Asura Khadga 2': '8C8A', // half-room cleave
+    'Asura Khadga 3': '8C8B', // half-room cleave
+    'Asura Khadga 4': '8C8C', // half-room cleave
+    'Asura Khadga 5': '8C8D', // half-room cleave
+    'Asura Khadga 6': '8C8E', // half-room cleave
+    'Asura Face of Wrath': '8CA6', // red/blue mechanic (red unsafe)
+    'Asura Face of Delight': '8CA7', // red/blue mechanic (blue unsafe)
+    'Asura Myriad Aspects 1': '8CB4', // wheel 2->1 dodge (first hit)
+    'Asura Myriad Aspects 2': '8CB5', // wheel 2->1 dodge (second hit)
```

Comment:
A tiny thing, but this might be better as “protean” or “fan” dodge. I don’t *think* we call this mechanic a “wheel” anywhere else.

Reply:
> A tiny thing, but this might be better as “protean” or “fan” dodge. I don’t _think_ we call this mechanic a “wheel” anywhere else.

That's what I was looking for, thanks.  Brain wasn't working 😄